integration/scripts/runCodegen.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
```

Comment:
Given the `scripts/` directory only has a single file in it, i'd be tempted to just nudge this up into the `integration/` directory directly.

Reply:
Sure - np.